docs/modules/DataSources.md

```diff
@@ -159,6 +159,59 @@ const testDataSource = {
 
 In this example, the `onRecordSetValue` transformer ensures that the `content` property is always an uppercase string.
 
+## Storing DataSources in Project JSON
+
+GrapesJS allows you to control whether a DataSource should be stored statically in the project JSON. This is useful for managing persistent data across project saves and loads.
+
+### Using the `shouldStoreInProject` Key
+
+When creating a DataSource, you can use the `shouldStoreInProject` key to specify whether it should be included in the project JSON.
+
+**Example: Creating a DataSource with `shouldStoreInProject`**
+
+```ts
+const persistentDataSource = {
+  id: 'persistent-datasource',
+  records: [
+    { id: 'id1', content: 'This data will be saved' },
+    { id: 'id2', color: 'blue' },
+  ],
+  shouldStoreInProject: true,
+};
+
+editor.DataSources.add(persistentDataSource);
+
+const temporaryDataSource = {
+  id: 'temporary-datasource',
+  records: [
+    { id: 'id1', content: 'This data will not be saved' },
+  ],
+  shouldStoreInProject: false, // This is the default if not specified
```

Comment:
It's nice to have the ability to skip some dataSources from the storage but I see it as more of an exception. In most of the cases, dataSource has to be saved, so I'd put `true` as default.
ps. the property could also be renamed in `skipFromStorage`

Reply:
Changed to store by default and usage of `skipFromStorage` now. 

* https://github.com/GrapesJS/grapesjs/pull/6160/commits/d35f56f7fff766f2a508b502dde1c86ff28139f9